### PR TITLE
Added initial documentation for ZGWs apis

### PR DESCRIPTION
**Pull request by ivo-ritense**
Includes:
- documenten-api
- zaken-api
- openzaak

### CODE REVIEW

File: using-valtimo/plugin/documenten-api/configure-plugin.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+The Documenten API plugin can be used to store documents and their metadata in an application that supports the
+Documenten API standard created and used by the Dutch government.
+
+## Configure the plugin
+
+In order to use this plugin a configuration has to be created. A general description on how to configure
+plugins can be found [here](../configure-plugin.md).
+
+If the Documenten API plugin is not visible in the plugin menu, it is possible the application is missing a dependency.
+Instructions on how to add the OpenZaak dependency can be found
```
Comment (Rick-Ritense) [inline using-valtimo/plugin/documenten-api/configure-plugin.md:12]: You mention the OpenZaak dependency while you talk about the visibility of the Documenten API plugin

File: using-valtimo/plugin/zaken-api/configure-plugin.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+Zaken API standard created and used by the Dutch government.
+
+## Configure the plugin
+
+In order to use this plugin a configuration has to be created. A general description on how to configure
+plugins can be found [here](../configure-plugin.md).
+
+If the Zaken API plugin is not visible in the plugin menu, it is possible the application is missing a dependency.
+Instructions on how to add the OpenZaak dependency can be found
+[here](../../../valtimo-implementation/modules/documenten-api.md)
```
Comment (Rick-Ritense) [inline using-valtimo/plugin/zaken-api/configure-plugin.md:13]: You describe the Zaken API visibility, mention the OpenZaak dependency and link the documenten-api.md

File: valtimo-implementation/modules/documenten-api.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# SmartDocuments
```
Comment (Rick-Ritense) [inline valtimo-implementation/modules/documenten-api.md:1]: The title is not correct

File: valtimo-implementation/modules/openzaak.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# SmartDocuments
```
Comment (Rick-Ritense) [inline valtimo-implementation/modules/openzaak.md:1]: The title is not correct

File: valtimo-implementation/modules/zaken-api.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+# SmartDocuments
```
Comment (Rick-Ritense) [inline valtimo-implementation/modules/zaken-api.md:1]: The title is not correct

File: introduction/modules/modules.md [hunk 6]
```diff
@@ -7,10 +7,28 @@
...
 services, such as transactional email and  generating documents. Some plugins are provided
 by the platform, but developers can create their own plugins as well. Plugins can be 
 configured without writing code, and can be used during BPMN processes. A more in-depth
 description can be found [here](plugin-introduction.md).
+
+## ZGW (Zaakgericht Werken)
+
+### Documenten API
+
+The Documenten (Documents) API is a Dutch government API standard meant to standardize the storage of and access to
```
Comment (marijnritense) [inline introduction/modules/modules.md:28]: A standard is already meant to standardize. Maybe rephrase this to:
The Documenten (Documents) API is a Dutch government API standard for storage of and access to...

File: introduction/modules/modules.md [hunk 7]
```diff
@@ -7,10 +7,28 @@
...
+## ZGW (Zaakgericht Werken)
+
+### Documenten API
+
+The Documenten (Documents) API is a Dutch government API standard meant to standardize the storage of and access to 
+documents. A plugin is provided that allows for connecting to applications that provide a Documenten API.
+
+### Zaken API
+
+The Zaken (Cases) API is a Dutch government API standard meant to standardize the information exchange around cases.
```
Comment (marijnritense) [inline introduction/modules/modules.md:33]: A standard is already meant to standardize. Maybe rephrase this to:
The Zaken (Cases) API is a Dutch government API standard for information exchange around cases...

File: using-valtimo/plugin/documenten-api/configure-plugin.md [hunk 8]
```diff
@@ -0,0 +1,53 @@
+# Using the Documenten API Plugin
+
+The Documenten API plugin can be used to store documents and their metadata in an application that supports the
```
Comment (marijnritense) [inline using-valtimo/plugin/documenten-api/configure-plugin.md:3]: Instead of **can be**, use **is**. This is what it is meant for. Also missing an example here, but not sure if this is really an issue.

File: using-valtimo/plugin/documenten-api/configure-plugin.md [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+
+In order to use this plugin a configuration has to be created. A general description on how to configure
+plugins can be found [here](../configure-plugin.md).
+
+If the Documenten API plugin is not visible in the plugin menu, it is possible the application is missing a dependency.
+Instructions on how to add the Documenten API dependency can be found
+[here](../../../valtimo-implementation/modules/documenten-api.md)
+
+To configure this plugin the following properties have to be entered:
+- The `Documenten API URL` property should contain the complete base URL of the Documenten API to connect to. This
```
Comment (marijnritense) [inline using-valtimo/plugin/documenten-api/configure-plugin.md:16]: Not sure how I feel about regular fields being displayed as code block. Maybe try a structure like
- **Document API URL**. Conttains the complete base URL of the Documenten API to connect to.

(And repeat for the rest)

File: using-valtimo/plugin/documenten-api/configure-plugin.md [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+A general description on how to create process links, can be found [here](../create-process-link.md).
+
+### Save document
+
+The `Save document` action takes a file from the local temporary storage and stores it in the Documenten API as a new 
+Document. The document to be stored is assumed to have a reference stored in a process variable. After storing the 
+document, the resulting document URL will be saved in a process variable.
+
+When creating a process link the following properties have to be entered:
+- `Name of process variable with document` The name of the process variable in which the ID of a local file in the
```
Comment (marijnritense) [inline using-valtimo/plugin/documenten-api/configure-plugin.md:42]: Same here

File: using-valtimo/plugin/documenten-api/configure-plugin.md [hunk 11]
```diff
@@ -0,0 +1,53 @@
+# Using the Documenten API Plugin
+
+The Documenten API plugin can be used to store documents and their metadata in an application that supports the
+Documenten API standard created and used by the Dutch government.
```
Comment (marijnritense) [inline using-valtimo/plugin/documenten-api/configure-plugin.md:4]: and is used

File: using-valtimo/plugin/openzaak/configure-plugin.md [hunk 12]
```diff
@@ -0,0 +1,24 @@
+# Using the OpenZaak Plugin
+
+The OpenZaak plugin can be used to provide authentication for ZGW (Zaakgericht Werken) plugins that connect to APIs which
```
Comment (marijnritense) [inline using-valtimo/plugin/openzaak/configure-plugin.md:3]: is used

File: using-valtimo/plugin/zaken-api/configure-plugin.md [hunk 13]
```diff
@@ -0,0 +1,47 @@
...
+
+### Link document to zaak
+
+The `Link document to zaak` action takes a Document stored in a Documenten API and link it to a zaak. The zaak to which 
+the Document is linked is based on the process for which the action is performed. The process belongs to a dossier that
+should already have a zaak instance link before this action is performed. The link between a zaak and a document is 
+stored as a `zaakinformatieobject` in the Zaken API.  
+
+When creating a process link the following properties have to be entered:
+- `URL to the document` The complete URL of the document in a Documenten API.
```
Comment (marijnritense) [inline using-valtimo/plugin/zaken-api/configure-plugin.md:40]: Same here

File: using-valtimo/plugin/documenten-api/configure-plugin.md [hunk 14]
```diff
@@ -0,0 +1,53 @@
...
+
+The Documenten API plugin supports the following actions that can be configured in process links in order to store 
+documents and their metadata. 
+
+A general description on how to create process links, can be found [here](../create-process-link.md).
+
+### Save document
+
+The `Save document` action takes a file from the local temporary storage and stores it in the Documenten API as a new 
+Document. The document to be stored is assumed to have a reference stored in a process variable. After storing the
```
Comment (marijnritense) [inline using-valtimo/plugin/documenten-api/configure-plugin.md:38]: A reference to the document has to be available as a process variable.

File: using-valtimo/plugin/openzaak/configure-plugin.md [hunk 15]
```diff
@@ -0,0 +1,24 @@
+# Using the OpenZaak Plugin
+
+The OpenZaak plugin can be used to provide authentication for ZGW (Zaakgericht Werken) plugins that connect to APIs which 
+are supported in openzaak. Currently the plugin that this plugin can provide authentication for are the following:
```
Comment (marijnritense) [inline using-valtimo/plugin/openzaak/configure-plugin.md:4]: Weird sentence

Comment (marijnritense) [reply using-valtimo/plugin/openzaak/configure-plugin.md:4]: This plugin provides authentication for the following plugin types:

File: using-valtimo/plugin/openzaak/configure-plugin.md [hunk 16]
```diff
@@ -0,0 +1,24 @@
...
+- [Zaken API](../zaken-api/configure-plugin.md)
+
+## Configuring the plugin
+
+In order to use this plugin a configuration has to be created. A general description on how to configure
+plugins can be found [here](../configure-plugin.md).
+
+If the OpenZaak plugin is not visible in the plugin menu, it is possible the application is missing a dependency.
+Instructions on how to add the OpenZaak dependency can be found 
+[here](../../../valtimo-implementation/modules/openzaak.md)
```
Comment (marijnritense) [inline using-valtimo/plugin/openzaak/configure-plugin.md:16]: Missing a dot at the end

File: using-valtimo/plugin/openzaak/configure-plugin.md [hunk 17]
```diff
@@ -0,0 +1,24 @@
...
+
+The OpenZaak plugin can be used to provide authentication for ZGW (Zaakgericht Werken) plugins that connect to APIs which 
+are supported in openzaak. Currently the plugin that this plugin can provide authentication for are the following:
+
+- [Documenten API](../documenten-api/configure-plugin.md)
+- [Zaken API](../zaken-api/configure-plugin.md)
+
+## Configuring the plugin
+
+In order to use this plugin a configuration has to be created. A general description on how to configure
```
Comment (marijnritense) [inline using-valtimo/plugin/openzaak/configure-plugin.md:11]: A plugin configuration is required before a plugin can be used.

File: using-valtimo/plugin/openzaak/configure-plugin.md [hunk 18]
```diff
@@ -0,0 +1,24 @@
...
+plugins can be found [here](../configure-plugin.md).
+
+If the OpenZaak plugin is not visible in the plugin menu, it is possible the application is missing a dependency.
+Instructions on how to add the OpenZaak dependency can be found 
+[here](../../../valtimo-implementation/modules/openzaak.md)
+
+The 'Client ID' and 'Secret' have to match these field as configured for the 'Applicatie' in OpenZaak.
+These can be found by going to 'API Autorisaties' &rarr; 'Applicaties' in the menu of the management interface of the 
+OpenZaak application. Here an existing application can be accessed to view the credentials, or a new one can be created
```
Comment (marijnritense) [inline using-valtimo/plugin/openzaak/configure-plugin.md:20]: A new 'applicatie' can be created, and existing 'applicaties' can be accessed to view credentials.

Comment (marijnritense) [reply using-valtimo/plugin/openzaak/configure-plugin.md:20]: Alternatively: Not our responsibility, so instead we can say it's somewhere in openzaak

File: using-valtimo/plugin/zaken-api/configure-plugin.md [hunk 19]
```diff
@@ -0,0 +1,47 @@
+# Using the Zaken API Plugin
+
+The Zaken API plugin can be used to store and access data about a Zaak (case) in an application that supports the
+Zaken API standard created and used by the Dutch government.
+
+## Configure the plugin
+
+In order to use this plugin a configuration has to be created. A general description on how to configure
```
Comment (marijnritense) [inline using-valtimo/plugin/zaken-api/configure-plugin.md:8]: A plugin configuration is required before a plugin can be used.

**Review by marijnritense — APPROVED** [review]
